services/video_chess_indexer/main/square.py

```diff
@@ -0,0 +1,17 @@
+from piece_id_lookups import id_to_short
+from piece_id_lookups import id_to_verbose
+
+class Square:
+    highlighted = False
+    piece_short = ''
+    piece_verbose = ''
```

Comment:
why class variables? these should be instance variables, no?
something like this maybe:
```
class Square:
    def __init__(self, piece_id):
        self.piece_id = piece_id
        self.init_square()

    def init_square(self):
        if self.piece_id == '12':
            self.highlighted = True
        else:
            self.highlighted = False
            self.piece_short = id_to_short[piece_id]
            self.piece_verbose = id_to_verbose[piece_id]
```

Reply:
Fixed